src/exampleTypes/Exists.js

```diff
@@ -2,14 +2,14 @@ import React from 'react'
 import F from 'futil'
 import { contexturify } from '../utils/hoc'
 
-let Exists = ({ tree, node, theme: { RadioList } }) => (
+let Exists = ({ tree, node, options=['exists', 'doesNotExist'], theme: { RadioList } }) => (
```

Comment:
This is basically the same component as bool. Wonder if it makes sense to extract the common stuff to one component and use it in both ... I did not do it since I can not imagine that was not discussed before ect.